backend/middleware/jwt.go

```diff
@@ -144,14 +135,6 @@ func JWTWebAuth(auth services.Auth) gin.HandlerFunc {
 
 			c.Next()
 			return
-		} else if ve, ok := err.(*jwt.ValidationError); ok {
-			if ve.Errors&jwt.ValidationErrorMalformed != 0 {
-				slog.Warn("That's not even a token")
-			} else if ve.Errors&(jwt.ValidationErrorExpired|jwt.ValidationErrorNotValidYet) != 0 {
-				slog.Warn("Token is either expired or not active yet")
-			} else {
-				slog.Warn("Couldn't handle this token", "error", err)
-			}
```

Comment:
Same reasoning as https://github.com/diggerhq/digger/pull/2279/files#r2394991072. These are not required anymore because the error handling at line 130-134 already handles it.

https://github.com/diggerhq/digger/blob/56160da22d17f41414aa9577cfee22ec0247fdab/backend/middleware/jwt.go#L123-L134